api/api-doc/task_manager.json

```diff
@@ -291,7 +291,7 @@
          "id":"task_status",
          "description":"A task status object",
          "properties":{
-            "id":{
+            "task_id":{
                "type":"string",
```

Comment:
Isn't this a public API? What about backward compatibility?

Reply:
Yes, it is. I also planned on dropping the fields that are unused by the task (like not showing keyspace if it isn't specified), but I guess I need to give up on both